.travis.yml

```diff
@@ -1,4 +1,3 @@
 language: go
 go:
   - 1.6
-  - tip
```

Comment:
The build box is reporting 1.6.3 is the latest version of Go installed.  Apparently, specifying `1.6` and `tip` cause Travis to use Go 1.6 and then Go 1.5.  Changes to Reign include using the newest ciphers from Go 1.6, precluding successful testing with older Go versions.